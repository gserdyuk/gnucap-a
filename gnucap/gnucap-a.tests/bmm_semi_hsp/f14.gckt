* test f14 - model with rsh, l, w; L - too small so GMAX is used;  bmm_semi_hsp


R1  1 0 rmodel L=1.e-12
.model rmodel r rsh=10 w=2 l=4 
* r= 1.e-12 / 2 * 10 = 5.e-12; resmin=1/1000/gmin; i=1/resmin = 1.e5
* indeed, as R(v1) also = 1.e-5, so total current = 50e-3 and V(1) =0.5

v1  1 0 dc 1

.print op  v(r1) I(v1) 
.op 

.end
