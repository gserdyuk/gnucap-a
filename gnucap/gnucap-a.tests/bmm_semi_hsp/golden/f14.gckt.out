Gnucap gnucap-arails  2009.12.07 RCS 26.136 master 2010-09-25  bmm_semi_hsp, bm_model fix, back to bmm_semi
The Gnu Circuit Analysis Package
Never trust any version less than 1.0
Copyright 1982-2009, Albert Davis
Gnucap comes with ABSOLUTELY NO WARRANTY
This is free software, and you are welcome
to redistribute it under the terms of 
the GNU General Public License, version 3 or later.
See the file "COPYING" for details.
* test f14 - model with rsh, l, w; L - too small so GMAX is used;  bmm_semi_hsp 
R1:  at rmodel: RS001 resistance is close to zero, resmin used
R1:  at rmodel: RS001 resistance is close to zero, resmin used
#           v(R1)      I(v1)     
 27.        0.5       -50.K      
